ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -39,29 +39,41 @@ def statistical_dtype_values(draw, *, function):
             length=size,
         )
     )
-    return dtype, values
+    shape = np.asarray(values, dtype=dtype).shape
+    axis = draw(helpers.get_axis(shape=shape, allow_none=True))
+    return dtype, values, axis
+
+
+@st.composite
+def statistical_correction_values(draw, *, function):
+    correction = draw(
+        st.integers()
+        | st.floats()
+    )
+    return correction
 
 
 # min
 @given(
-    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_numeric_dtypes),
+    dtype_and_x=statistical_dtype_values(function="min"),
     num_positional_args=helpers.num_positional_args(fn_name="min"),
     data=st.data(),
 )
 @handle_cmd_line_args
 def test_min(
     *,
-    data,
     dtype_and_x,
-    as_variable,
-    with_out,
     num_positional_args,
-    native_array,
-    container,
-    instance_method,
+    data,
     fw,
 ):
-    input_dtype, x = dtype_and_x
+    input_dtype, x, axis = dtype_and_x
+    as_variable = data.draw(st.booleans())
+    with_out = data.draw(st.booleans())
```

Comment:
This is not needed, we should keep these arguments in the argument list but not the `@given` decorator, the `@handle_cmd_line_args` wrapper will add them.

Reply:
This has been removed, and the necessary changes have been made.